draft-ietf-quic-applicability.md

```diff
@@ -446,6 +446,17 @@ leakage of information about the data that is sent. A QUIC implementation can
 expose an interface that allows an application layer to specify how to apply
 padding.
 
+# ACKing on constraint links
+
+On network paths where there is significant asymmetry,
+the transmission of QUIC ACKs can reduce the efficient use of network
+capacity or transmission opportunities. This effect occurs
```

Comment:
```suggestion
capacity. This effect occurs
```
This sentence is still a bit awkward, but I'm not sure what else to suggest.

Reply:
... Radio technologies can have costs per-packet, not per byte - and a sending packet high rate impacts that... in ways for QUIC that would not happen currently with TCP. 
I suggest a new additional sentence, something like:
"On some types of link, the rate of transmission of QUIC packets carrying control information can also impact the overall link efficiency and ultimately the user experience."